frontend/talentsearch/src/js/components/experienceAndSkills/ExperienceAndSkills.tsx

```diff
@@ -68,7 +68,9 @@ export const compareByDate = (e1: ExperienceForDate, e2: ExperienceForDate) => {
   return e2EndDate - e1EndDate;
 };
 
-const flattenExperienceSkills = (experiences: MergedExperiences): Skill[] => {
+export const flattenExperienceSkills = (
```

Comment:
Same as `getMissingSkills`, I can see moving this to helpers as beneficial.

Reply:
Fixed by [Move reusable functions into helper folders](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3863/commits/3aae2129d3f34a9a21bcf9b33d6a421e3e73480d)